lms/views/dashboard/api/course.py

```diff
@@ -28,7 +28,7 @@ class CoursesMetricsSchema(PyramidRequestSchema):
     assignment_ids = fields.List(fields.Integer(), data_key="assignment_id")
     """Return metrics for these assignments only."""
 
-    courses_ids = fields.List(fields.Integer(), data_key="course_id")
+    course_ids = fields.List(fields.Integer(), data_key="course_id")
```

Comment:
Typo. It is read as `course_ids` later in the action method.

Since `course_ids` is more consistent with `assignment_ids`, I corrected it here rather than the other way around.